mappings/net/minecraft/util/SegmentedAnglePrecision.mapping

```diff
@@ -1,3 +1,22 @@
 CLASS net/minecraft/unmapped/C_bhznmlir net/minecraft/util/SegmentedAnglePrecision
+	FIELD f_errgtmjh precision I
+	FIELD f_mqmrqvqi degreeToAngle F
```

Comment:
```suggestion
	FIELD f_mqmrqvqi degreesToAngle F
```
or is there some reason this and the following fields use the singlular `degree`?